crates/subspace-proof-of-space/benches/pos.rs

```diff
@@ -23,14 +25,29 @@ fn pos_bench<PosTable>(
 ) where
     PosTable: Table,
 {
+    ThreadPoolBuilder::new()
```

Comment:
I think you're missing a` #[cfg(feature = "parallel")]` here?
It fails to compile `subspace-proof-of-space` separately:
```
error[E0433]: failed to resolve: use of undeclared type `ThreadPoolBuilder`
  --> crates/subspace-proof-of-space/benches/pos.rs:28:5
   |
28 |     ThreadPoolBuilder::new()
   |     ^^^^^^^^^^^^^^^^^ use of undeclared type `ThreadPoolBuilder`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `subspace-proof-of-space` (bench "pos") due to previous error
```

Reply:
And if I bench anyway (`cargo bench --features chia --bench pos`) it panics
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ThreadPoolBuildError { kind: GlobalPoolAlreadyInitialized }', crates/subspace-proof-of-space/benches/pos.rs:32:10
```

Reply:
Fixed, though I'm not even sure if it should be limited to 4 cores by default (I wanted to have more representative results, using all cores would be faster).